### ISSUE TITLE
Review design of char table 2.7

### ISSUE THREAD

**Issue by daobrien** [StyleGuides/WritingStyleGuide#366]
https://stylepedia.net/style/5.1/#punc-names

The layout of table 2.7 is not great. It's not terrible, but I think it could be better. Please offer suggestions.

@style-guide-committers
@PatSharp99

**Comment by julian-cable** [comment]
The first table column should be narrower. @daobrien, if I recall correctly, you and I spent some time on it before the last release without figuring out a good way to achieve it with valid markup. Any suggestions are welcomed.

**Comment by daobrien** [comment]
Yeah I was thinking narrower and maybe not left justified (or maybe with more char space). The table itself probably doesn't need to be full width, either.

I don't remember looking at this table before, but changing the col widths in DocBook is easy enough. E.g., you can do something like this to get col2 twice the width of col1:
~~~
<colspec colnum="1" colname="col1" colwidth="1*"/>
<colspec colnum="2" colname="col2" colwidth="2*"/>
~~~
https://tdg.docbook.org/tdg/4.5/table.html has all the info about how to lay out tables, it's just a matter of deciding how we want to lay it out.

**Comment by daobrien** [comment]
This is also tagged as 6.0 but not assigned, and no PR visible. @julian-cable can you maybe go through and sort out the milestones so we know where we stand? Thanks.

**Comment by julian-cable** [comment]
I have reviewed the milestones and removed 6.0 from those issues that will not be covered in this release. 

I still want to try to cover this item if I can: https://github.com/StyleGuides/WritingStyleGuide/issues/351
